src/main/java/no/unit/nva/model/Contributor.java

```diff
@@ -1,27 +1,43 @@
 package no.unit.nva.model;
 
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.exceptions.MalformedContributorException;
 
 import java.util.List;
 import java.util.Objects;
 
+import static java.util.Objects.isNull;
+
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 public class Contributor {
 
+    public static final String CORRESPONDING_AUTHOR_EMAIL_MISSING =
+            "The Contributor is corresponding author, but no email for correspondence is set";
```

Comment:
So email may be set for all contributors but is obligatory for corresponding authors?

Reply:
Yes